### ISSUE TITLE
Improve "Features" component 

### ISSUE THREAD

**Issue by sattler9** [ManifoldScholar/manifold#2880]
## Feature Description

I would like to be able to style the feature a bit more, especially the color of each of the text parts, including the sign-up button. 
In particular, I'm trying to change:
1. The sign-up button's color and I'd love to be able to wrap it in a background class to make it look like a button
2. the color of the "Link text" field in a feature
3. the color of a linked text when a mouse hovers over it

## *Why* is this feature important? *Who* does it help?

Right now, I'm unable to use the sign-up button because the color of the background for my feature (banner) is super dark (#18453b) and the sign-up button text is also dark. I am unable to change it without changing the theme's accent color. I'm not sure how that works, so it makes it difficult to select a color that works on light and dark backgrounds and has an accessible color for the sign-up button. 

Basically, I'm looking to change the colors so they have enough contrast so that they pass accessibility testing but still match my institutional colors.

## User Stories

As a Manifold site admin, I want to be able to easily customize the look of the features banner so that I can provide better accessibility of the content.
